future_challenges_for_RSEng.tex

```diff
@@ -162,9 +162,19 @@ \section{Introduction}
 Challenges and Risks for {RSEng}'': \url{https://events.hifis.net/event/1741/contributions/14026/}},
 without claiming to represent the entire deRSE community.
 The material presented here is based on the output of the workshop participants, but extended with the authors' ideas.
-Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers,
-but no list or profile of the participants is available.
+Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers, but no list or profile of the participants is available.
 The workshop hosts felt, that the majority of participants were from the German RSE community.
+
+The workshop was held as an open discussion of 45 minutes between the participants.
+As can be seen from the workshop material~\cite{goth2025_WSPad}, we prepared some slides to steer the discussion
+towards the time frame of 2035.
+For a start we prepared questions that we openly asked to the audience,
+while also allowing the participants to meander towards questions of their interest.
+The prepared questions stem from internal discussions among the workshop hosts.
+It started from discussions whether the recently defined competencies of RSEs~\cite{Goth2024}
+are just snapshots of our time, or whether they can constitute something that stands the test of time.
+Then we asked ourselves how the community of RSEs would interact with some emerging trends.
+
```

Comment:
The "45' discussion" is a bit too short and also not the complete truth. We should further clarify:

```suggestion
The collected material was further discussed among the authors in follow-up meetings, structuring and refining the ideas, adding new ideas as described in each of the following sections.

```